assets/data/cards_pt-br.json

```
@@ -29,6 +29,13 @@
       ],
       "description": "Costumamos chamar de ambiente o lugar ou os servidores onde o que foi desenvolvido fica disponível para testes, validações, pessoas usuárias finais utilizarem ou demais necessidades. Esse servidor seria como um computador. Pode ser um computador físico ou disponível na nuvem. Existem os ambientes local e dev (onde o desenvolvimento está acontecendo), homolog (ou ainda hom, homol, homologação - onde os testes costumam ser feitos)e produção (onde está disponível o sistema final, desenvolvido e testado, para pessoas usuárias)."
     },
+    {
+      "title": "AIoT",
+      "tags": [
+        "Conceito"
+      ],
+      "description": "AIoT (Artificial Intelligence of Things), ou Inteligência Artificial das Coisas, constitui a combinação entre Inteligência Artificial e Internet das Coisas. Um de seus principais objetivos é o de potencializar o processamento de dados provenientes dos dispositivos de IoT de forma automatizada, mais rápida e com o mínimo de intervenção do ser humano. Isso agiliza a análise de dados e o processo de tomada de decisões, visto que sem a AI, haveria um grande volume de informação, mas com a dependência da mão de obra do ser humano para que ele se tornasse relevante para uso."
```

Deixei um pouco mas direto e didático aqui, o que acha?

```suggestion
      "description": "AIoT (Artificial Intelligence of Things), ou Inteligência Artificial das Coisas, é a combinação de Inteligência Artificial (IA) com a Internet das Coisas (IoT). Seu principal objetivo é automatizar e acelerar o processamento dos dados gerados pelos dispositivos de IoT, com o mínimo de intervenção humana. Isso facilita a análise dos dados e torna a tomada de decisões mais rápida. Sem a IA, embora houvesse um grande volume de informações disponíveis, seria necessário depender da intervenção humana para torná-las úteis."
```

---

Acho que ficou melhor, obrigado 🌱